csharp/ql/test/library-tests/controlflow/graph/NodeGraph.expected

```diff
@@ -2775,7 +2775,7 @@ Assert.cs:
 #-----| true -> access to parameter b2
 
 #  140| [assertion failure] access to parameter b2
-#-----| false -> [assertion failure] access to parameter b3
+#-----| true -> [assertion failure] access to parameter b3
```

Comment:
This change is because the stubs are still based on .NET 5. When we [upgraded to .NET 6](https://github.com/github/codeql/pull/7784/files#diff-17dfae8530cd04aeb538adfee48b5875d42448215250b2a6aef6562beb1e68f0R2778), this line changed.